docs/build_docs.py

```diff
@@ -98,6 +102,12 @@ def parse_args_with_argparse():
         default=None,
         help="Output dir to write built docs to",
     )
+    parser.add_argument(
+        "--save-temp-dir",
+        type=str,
+        default=None,
+        help="Output dir to write temporary build files to (for debugging purposes)",
+    )
```

Comment:
@biosafetylvl5 I expect you will want to just look at the changes I made here, and do something similar to your current iteration of build_docs.py on the integration branch.  Probably easier than trying to truly resolve conflicts / merge the changes.  I made pretty minimal updates.